tests/pr_test.go

```diff
@@ -19,6 +19,15 @@ const quickstartExampleTerraformDir = "examples/quickstart"
 const roksPatternTerraformDir = "patterns/roks"
 const resourceGroup = "geretain-test-resources"
 
+// Temp: Will be updated once issue https://github.ibm.com/GoldenEye/issues/issues/4302 is fixed
```

Comment:
lets be clear here - the `atracker_target` ignore is being tracked in https://github.ibm.com/GoldenEye/issues/issues/4302

The ACL ignores can be removed once we merge this PR

Reply:
ok, sure